fix: Add examples for sensitive fields to docs (#5405)
getsentry/sentry-docs: src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx

  - github_token
  - privatekey
  - private_key

> - Values that contain strings in, or whose keynames are listed in, Sensitive Fields in your Project Settings.

Reviewer: Do you mean the "Additional Sensitive Fields" field in [Project] > Settings > Security & Privacy, or something else?
```suggestion
- Values that contain strings in, or whose keynames are listed in, [Project] > Settings > Security & Privacy in "Additional Sensitive Fields".
```

Author: yeah that 👍

>   - An entry in Sensitive Fields such as `mysekret` will cause removal of any field named `mysekret`, but also removes any field _value_ that contains `mysekret`. Sentry does this to protect against sensitive data leaking as part of structured data that has been sent as a single string to Sentry (such as, a JSON object that is stringified and embedded as JSON string in another JSON structure).

Reviewer: ```suggestion
  - An entry in "Additional Sensitive Fields" such as `mysekret`, for example, will cause the removal of any field named `mysekret`, but also removes any field _value_ that contains `mysekret`. Sentry does this to protect against sensitive data leaking as part of structured data that has been sent as a single string to Sentry (such as, a JSON object that is stringified and embedded as JSON string in another JSON structure).
```

>   - As an extreme example where this behavior can become surprising, the string `"Unexpected error"` will be removed from events if there is an entry `exp` in Sensitive Fields.

Reviewer: ```suggestion
  - As an extreme example where this behavior can become surprising, the string `"Unexpected error"` will be removed from events if the entry `exp` is in "Additional Sensitive Fields".
```
